### ISSUE TITLE
Feature/Progress

### ISSUE THREAD

**Issue by Snugug** [IBM-Watson/patterns#63]
Progress Bar element

**Comment by una** [comment]
Will we need js implemented/architected before tackling this?

**Comment by Snugug** [comment]
Nope, it's an HTML5 element. We can style it w/o needing JavaScript to update the values and we can provide styled support for [a fallback](https://css-tricks.com/html5-progress-element/) when the progress element isn't available. It'd be nice to provide a simple JS API for updating both, but not required for basic styling.

That being said, it's low priority so I'm moving it to the RC1 milestone
